### PR TITLE
{data}[fosscuda/2019b] pycocotools v2.0.1 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/pycocotools/pycocotools-2.0.1-fosscuda-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+homepage = 'https://github.com/cocodataset/cocoapi'
+description = "Tools for working with the MSCOCO dataset"
+
+toolchain = {'name': 'fosscuda', 'version': '2019b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['1c06e73a85ed9874c1174d47064524b9fb2759b95a6997437775652f20c1711f']
+
+dependencies = {
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/p/pycocotools/pycocotools-2.0.1-fosscuda-2019b-Python-3.7.4.eb:16]: a set also works but this is typically a list :)

**Review by migueldiascosta — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/p/pycocotools/pycocotools-2.0.1-fosscuda-2019b-Python-3.7.4.eb (hunk 1 above)
Comment (smoors) [reply easybuild/easyconfigs/p/pycocotools/pycocotools-2.0.1-fosscuda-2019b-Python-3.7.4.eb]: well spotted :)
fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/11211/commits/f311672ece41811e22d332d35168bdf3395c0f1b

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node371.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/c0d36384278047266c3c88c139c781a7 for a full test report.

**Review by migueldiascosta — APPROVED** [review]
lgtm

**Comment by migueldiascosta** [comment]
Going in, thanks @smoors!

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 2 out of 12 (1 easyconfigs in this PR)
nuc.lan - Linux Fedora 32, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.8.5
See https://gist.github.com/9588cd3074e09f5cb71b54b0ccc1bf98 for a full test report.

**Pull request merged**